monitoring/config.json

```diff
@@ -0,0 +1,17 @@
+{
+    "input_folder": "model_monitoring",
+    "reference_file_name": "reference_data.csv",
```

Comment:
I've added the sample dataset into the repo. The template assumes that the data is stored in Azure ML (in this case, as csv file but the data scientists can change that to match the correct format of the data).
I kept it like this because the data collection process hasn't been designed yet. Once we know more about that process, the template can be modified to read directly from the relevant data sources (Cosmos DB, Feature store etc.)